--- last.fm-1.4.2.58240/src/container.cpp.old	2008-05-11 00:13:16.000000000 +0200
+++ last.fm-1.4.2.58240/src/container.cpp	2008-05-11 00:13:50.000000000 +0200
@@ -519,7 +519,7 @@
     // Hack to get it working with VS2005
     QString path = qApp->applicationDirPath();
   #else
-    QString path = qApp->applicationDirPath() + "/extensions";
+    QString path = "@LIB@/lastfm-radio/extensions";
   #endif
 
   #ifndef QT_NO_DEBUG
--- last.fm-1.4.2.58240/src/libMoose/MooseCommon.cpp.old	2008-05-11 00:14:12.000000000 +0200
+++ last.fm-1.4.2.58240/src/libMoose/MooseCommon.cpp	2008-05-11 00:16:03.000000000 +0200
@@ -43,7 +43,7 @@
 QString
 dataPath( QString file )
 {
-    return QApplication::applicationDirPath() + "/data/" + file;
+    return "@LIB@/lastfm-radio/data/" + file;
 }
 
 
@@ -108,7 +108,7 @@
         // Hack to get it working with VS2005
         dirPath = qApp->applicationDirPath();
     #else
-        dirPath = qApp->applicationDirPath() + "/services";
+        dirPath = "@LIB@/lastfm-radio/services";
     #endif
 
     #ifndef QT_NO_DEBUG
