diff -ur last.fm-1.3.0.58-o/src/container.cpp last.fm-1.3.0.58/src/container.cpp
--- last.fm-1.3.0.58-o/src/container.cpp	2007-06-22 12:21:53.000000000 -0600
+++ last.fm-1.3.0.58/src/container.cpp	2007-07-21 12:17:37.000000000 -0600
@@ -977,7 +977,7 @@
     // Hack to get it working with VS2005
     dirPath = qApp->applicationDirPath();
   #else
-    dirPath = qApp->applicationDirPath() + "/extensions";
+    dirPath = "/usr/lib/lastfm-radio/extensions";
   #endif
 
   #ifndef QT_NO_DEBUG
diff -ur last.fm-1.3.0.58-o/src/libLastFmTools/containerutils.cpp last.fm-1.3.0.58/src/libLastFmTools/containerutils.cpp
--- last.fm-1.3.0.58-o/src/libLastFmTools/containerutils.cpp	2007-06-22 12:21:53.000000000 -0600
+++ last.fm-1.3.0.58/src/libLastFmTools/containerutils.cpp	2007-07-21 12:18:48.000000000 -0600
@@ -67,7 +67,7 @@
 
 QString dataPath( QString file )
 {
-    return QApplication::applicationDirPath() + "/data/" + file;
+    return QString("/usr/share/lastfm-radio/data/") + file;
 }
 
 
@@ -76,7 +76,7 @@
     QString prefix;
     QString midfix;
 
-    prefix = QApplication::applicationDirPath();
+    prefix = QString("/usr/share/lastfm-radio");
     #ifdef WIN32
         if ( ( QSysInfo::WindowsVersion & QSysInfo::WV_DOS_based ) == 0 )
         {
@@ -102,7 +102,7 @@
         prefix = QDir::home().filePath( ".local/share/Last.fm" );
     
     #else
-		prefix = QApplication::applicationDirPath();
+		prefix = QString("/usr/share/lastfm-radio");
     #endif
 
     QDir d( prefix );
@@ -119,7 +119,7 @@
         // Hack to get it working with VS2005
         dirPath = qApp->applicationDirPath();
     #else
-        dirPath = qApp->applicationDirPath() + "/services";
+        dirPath = QString("/usr/lib/lastfm-radio/services");
     #endif
 
     #ifndef QT_NO_DEBUG
Only in last.fm-1.3.0.58/src/libLastFmTools: containerutils.cpp~
