--- lastfm-player/src/player.cpp.orig	2006-02-08 12:59:21.000000000 +0100
+++ lastfm-player/src/player.cpp	2006-02-08 14:28:16.000000000 +0100
@@ -18,6 +18,7 @@
  *   51 Franklin Steet, Fifth Floor, Boston, MA  02111-1307, USA.          *
  ***************************************************************************/
 
+#include <QDir>
 #include <QtGui>
 #include <QHttp>
 #include <QFileInfo>
@@ -66,7 +67,7 @@
     , m_tagDialog( new TagDialog( this ) )
     , m_playback( new Playback() )
     , m_timer( new QTimer() )
-    , m_coverLoader( new CoverLoader( this, QApplication::applicationDirPath() + "/cache/" ) )
+    , m_coverLoader( new CoverLoader( this, QDir::homeDirPath() + "/.last.fm/cache/" ) )
     , m_coverBuffer( new QBuffer() )
     , m_animation( new QMovie( QApplication::applicationDirPath() + "/data/progress_active.mng" ) )
     , m_proxyConnected( false )
